rust/src/lib.rs

```diff
@@ -204,6 +204,17 @@ mod ffi {
         fn cache_branch_to_nevra(nevra: &str) -> String;
     }
 
+    unsafe extern "C++" {
+        include!("rpmostree-util.h");
+        // Currently only used in unit tests
+        #[allow(dead_code)]
```

Comment:
Minor: I think a `#[cfg(test)]` attribute would be the correct way to handle this.